dir.targets

```diff
@@ -102,6 +113,16 @@
     <Exec Condition="'$(OsEnvironment)'=='Unix'"
           Command="find '$(RoslynPackageDir)tools' -name &quot;*.exe&quot; -exec chmod &quot;+x&quot; '{}' ';'" />
 
+    <!-- Download latest dotnet -->
+    <DownloadFile FileName="$(DotnetPackagePath)"
+                  Address="https://dotnetcli.blob.core.windows.net/dotnet/dev/Binaries/Latest/dotnet-win-x64.latest.zip"
```

Comment:
Why would we want to configure this? If we decide to change it, we can update the script directly. In which instances out one person use one versus another uri?